.github/workflows/backend-integration-test.yml

```diff
@@ -132,11 +128,7 @@ jobs:
         architecture: [ linux/amd64 ]
         java-version: [ 17 ]
         test-mode: [ embedded, deploy ]
-        include:
-          - test-mode: 'embedded'
-            backend: 'h2'
-          - test-mode: 'deploy'
-            backend: 'mysql'
+        backend: [ h2, mysql, postgresql ]
```

Comment:
@mchades , I 've reverted your change here; no better solution has been found. Please take a look if you are free.

Reply:
you can use `exclude`, FYI: https://docs.github.com/en/enterprise-cloud@latest/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow#excluding-matrix-configurations

Reply:
I already checked it and it's a bit complicated for use to configure accurately, let me think about it.

Reply:
try this:
```
matrix:
        architecture: [linux/amd64]
        java-version: [ 17 ]
        test-mode: [embedded, deploy]
        backend: [h2, mysql, postgresql]
        exclude:
          - test-mode: 'embedded'
            backend: 'mysql'
          - test-mode: 'embedded'
            backend: 'postgresql'
          - test-mode: 'deploy'
            backend: 'h2'
```

Reply:
Done.